AE-1577 Sum of two heat losses instead of only one

muep opened the pull request:
The 2018 ET has two values for describing heat losses related the heat
distribution system:
- [:lahtotiedot :lammitys :tilat-ja-iv :lampohavio-lammittamaton-tila]
- [:lahtotiedot :lammitys :lammin-kayttovesi :lampohavio-lammittamaton-tila]

Before this change, only the first one of these was placed in the A9
cell of the 8th page of the ET PDF. This change adjusts it so that
their sum is used instead. A10 still gets the latter value.

muep committed:
AE-1577 Sum of two heat losses instead of only one

The 2018 ET has two values for describing heat losses related the heat
distribution system:
- [:lahtotiedot :lammitys :tilat-ja-iv :lampohavio-lammittamaton-tila]
- [:lahtotiedot :lammitys :lammin-kayttovesi :lampohavio-lammittamaton-tila]

Before this change, only the first one of these was placed in the A9
cell of the 8th page of the ET PDF. This change adjusts it so that
their sum is used instead. A10 still gets the latter value.

muep commented on etp-backend/src/main/clj/solita/etp/service/complete_energiatodistus.clj:
```diff
@@ -490,7 +490,12 @@
+                        [:toteutunut-ostoenergiankulutus :kaukojaahdytys-vuosikulutus-yhteensa-nettoala])
+          (combine-keys +
+                        0
+                        [:tulokset :lampohavio-lammittamaton-tila]
```
Tämän polun keksin omasta päästä. Saa ehdottaa paremmin nimeämiskäytäntöön sopivaa paikka tai nimeä.

hkorpi replied:
kyllä toi varmaan pitäisi pitää lähtötietojen puolella eli: `[:lahtotiedot :lammitys :lampohavio-lammittamaton-tila]`

hkorpi replied:
vai näytetääkö tämä tulosten puolella pdf:ssä?

hkorpi replied:
niin sehän taidetaan näyttää siis kohdassa: "Energiatodistuksen laatimisessa käytettyjä lähtötietoja" Eli silloin kuuluu lähtötietoihin ja lämmityksen lähtötietoihin.

muep replied:
No niinpäs onkin lähtötiedoksi kutsuttu tuolla PDF:ssa. Vaihdetaan se sinne lokeroon myös rajapinnassa.

muep commented on etp-backend/src/main/clj/solita/etp/service/energiatodistus_pdf.clj:
```diff
@@ -539,7 +539,7 @@
                {:path [:lahtotiedot :lammitys :lammin-kayttovesi :lampopumppu-tuotto-osuus] :dp 0 :percent? true}
-               {:path [:lahtotiedot :lammitys :tilat-ja-iv :lampohavio-lammittamaton-tila] :dp 1}
+               {:path [:tulokset :lampohavio-lammittamaton-tila] :dp 1}
                {:path [:lahtotiedot :lammitys :lammin-kayttovesi :lampohavio-lammittamaton-tila] :dp 1}]
```
Tämä A10 -soluun menevä arvo ei ole missään käytössä, en ihan tiedä kannattaisiko se jättää näin tai esimerkiksi poistaa kokonaan.

muep replied:
Vaihdettiin käyttöön sama arvo kuin ylemmässä solussa. xlsx:aan merkittiin kuitenkin UNUSED-nimellä

muep commented:
Tämä kivasti jo toimii ilman että on koskenut PDF:aan, mutta sieltä voisi A9 -solusta korjata placeholderin vastaamaan muutosta.

muep committed:
AE-1577 Rename the sum of heat losses

muep committed:
AE-1577 Add the sum of heat losses to private CSV

muep committed:
AE-1577 Fill in sum of heat loss twice

Hopefully it looks a bit less confusing this way.

muep committed:
AE-1577 Update placeholders in ET templates

In the 8th page of each template:
- A9 to [:lahtotiedot :lammitys :lampohavio-lammittamaton-tila]
- A10 to UNUSED

muep committed:
Revert "AE-1577 Add the sum of heat losses to private CSV"

This reverts commit 2bfe158d9a1430b6da86bc4a1f32463d5e947257.

Instead, we will fix the test so that it does not insist that this
gets exported.

muep committed:
AE-1577 Fix energiatodistus-csv-test

It is OK to not export the newly added attribute
